### PR TITLE
Add Regex Inspector and Email Inspector example.

### PR DESCRIPTION
**Pull request by MooooCat**
## Types of changes

- [ ] Maintenance (no change in code, maintain the project's CI, docs, etc.)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: LICENSE
Comment (Wh1isper) [inline LICENSE:1]: Any reason to change this file? There is already "Version 2.0" in text..

Comment (MooooCat) [reply LICENSE:1]: The `sweep`  robot change it, it act so weird lately 😂

I will discard these change.

**Review by Z712023 — APPROVED** [review]

**Comment by Wh1isper** [comment]
Can we test all default `Inspector` in https://github.com/hitsz-ids/synthetic-data-generator/blob/main/tests/manager/test_inspector_manager.py to verify that the default behaviour is normal and to ensure minimum availability.

**Comment by MooooCat** [comment]
> Can we test all default `Inspector` in https://github.com/hitsz-ids/synthetic-data-generator/blob/main/tests/manager/test_inspector_manager.py to verify that the default behaviour is normal and to ensure minimum availability.

Ah, using manager is a good idea. （妙啊）

**Pull request merged**